chatto/message.py

```diff
@@ -37,22 +38,48 @@
 from chatto.channel import Channel
 from chatto.stream import Stream
 
+class MessageTypes(enum.Enum):
+    ChatEndedEvent = "chatEndedEvent"
+    MessageDeletedEvent = "messageDeletedEvent" 
+    NewSponsorEvent = "newSponsorEvent"
+    SponsorOnlyModeEndedEvent = "sponsorOnlyModeEndedEvent"
+    SponsorOnlyModeStartedEvent = "sponsorOnlyModeStartedEvent"
+    MemberMilestoneChatEvent = "memberMilestoneChatEvent"
+    SuperChatEvent = "superChatEvent"
+    SuperStickerEvent = "superStickerEvent"
+    TextMessageEvent = "textMessageEvent"
+    Tombstone = "tombstone"
+    UserBannedEvent = "userBannedEvent"
 
 @dataclass(eq=True, frozen=True)
 class Message:
     id: str
-    type: str
+    type: MessageTypes
     stream: Stream
     channel: Channel
     published_at: dt.datetime
     content: str
 
+    def get_type(type: str) -> MessageTypes:
```

Comment:
make this a function cus yt api returns a str for the api type... this func is converting it to the enum